docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-infrastructure.adoc

```diff
@@ -90,282 +111,743 @@ Use the changelog.md file for a cumulative list of all policies that are added t
+
+As a security best practice, it is always recommended to use the latest version of GKE.
+
+Note: This Policy is in line with the https://cloud.google.com/kubernetes-engine/docs/release-schedule#schedule-for-release-channels[GCP GKE release] version schedule 
+
+*Policy Severity—* Medium
+
+*Policy Type—* Config
+
+*Impact—* Low. Open Alerts will be resolved for the GKE clusters where the GKE major and minor versions are not end of life as per the GCP release schedule.
+
+*Current RQL—*
+
+----
+config from cloud.resource where cloud.type = 'gcp' AND api.name = 'gcloud-container-describe-clusters' AND json.rule = isNodeVersionSupported exists AND isNodeVersionSupported does not equal "true"
+----
+
+*Proposed RQL—*
+
+----
+config from cloud.resource where cloud.type = 'gcp' AND api.name = 'gcloud-container-describe-clusters' AND json.rule = NOT ( currentNodeVersion starts with "1.27." or currentNodeVersion starts with "1.28." or currentNodeVersion starts with "1.29." or currentNodeVersion starts with "1.30." )
+----
+
+2+|*Policy Name and Metadata Update*
+
+|*Google Workspace User not enrolled with 2-step verification*
+
+//RLP-146693
+
+|*Changes—* New Policy Name: GCP Google Workspace User not enrolled with 2-step verification
+
+Policy metadata updated to provide more descriptive content.
+
+*Severity—* Medium
+
+*Policy Type—* Config
+
+*Impact—* None. No impact on alerts.
+
+|*Google Workspace Super Admin not enrolled with 2-step verification*
+
+//RLP-146694
+
+|*Changes—* New Policy Name: GCP Google Workspace Super Admin not enrolled with 2-step verification
+
+Policy name and metadata updated to provide more descriptive content.
+
+*Severity—* High
+
+*Policy Type—* Config
+
+*Impact—* None. No impact on alerts.
+
+|===
+
+[#cna-policy-updates]
+=== CNA Policy Updates
+
+
+The Policy Name, Description and Recommendation for Remediation steps have been revised to better align with the behavior of the following Cloud Network Analyzer (CNA) policies.
 
-The following IAM OOTB policies will be updated in 24.7.3 release.
 
 [cols="50%a,50%a"]
 |===
 |*Policies*
 |*Description*
+//RLP-146591
 
-|*AWS Users and Machine Identities with Excessive Policies*
-//RLP-145657
+|*AWS EC2 instance with network path from the internet (0.0.0.0/0)*
 
-|*Changes—* The policy RQL will be updated to narrow down the resource based policies and remove false positives.
+|*Description—* This policy identifies AWS EC2 instances with network path from the internet (0.0.0.0/0). AWS EC2 instances with network path from the internet increases the risk of unauthorized access, cyber attacks, and data breaches, as it may provide a larger attack surface for malicious actors. Such instances are especially prone to brute force or vulnerability exploits.
```

Comment:
You can remove "*Description—*" from all the policies as the column name itself is "Description"